recipes-devtools/qemu/qemu_%.bbappend

```diff
@@ -0,0 +1,5 @@
+# This temporary override enables compatibility of older openembedded-core
+# with newer meta-virtualization + xenclient-oe.
+# When OpenXT's openembedded-core is uprev'd to a version that already includes this
+# defintion in meta/recipes-devtools/qemu/qemu.inc then this can be removed here.
+PACKAGECONFIG[xen] = "--enable-xen,--disable-xen,xen-tools,xen-tools-libxenstore xen-tools-libxenctrl xen-tools-libxenguest"
```

Comment:
Is this change to qemu needed?  We use qemu-dm and qemu-dm-stubdom for our device emulators.

Reply:
Ack, I don't think it's needed. An incremental build without it works fine, so I'll drop this commit.